### DISCUSSION TITLE
How can I sort GeoJSON? For example, I want to query the list of people near the current user

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#14017] [Archive]
### Describe the Bug

I didn't find any documentation or examples of this

### To Reproduce

no

### What version of Directus are you using?

9.12.2

### What version of Node.js are you using?

18.0.0

### What database are you using?

Postgres 14

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by br41nslug** [comment]
There is currently no built-in way of sorting by distance without custom queries (implementing either the haversine algorithm using custom Postgis/Spatialite functions) as fare as I'm aware. There currently 4 filters built-in for geometric shapes like `_intersects`, `_intersects_bbox` but im not sure if these can do the query you want.

**Reply by ghost** [reply]
_intersects and intersects_bbox can only query some users within a fixed range, but cannot sort it by distance, sort only supports the sorting of fields, not the sorting of positions.
